controllers/sandbox_controller.go

```diff
@@ -280,6 +298,38 @@ func (r *SandboxReconciler) reconcilePod(ctx context.Context, sandbox *sandboxv1
 	return pod, nil
 }
 
+func (r *SandboxReconciler) reconcilePVCs(ctx context.Context, sandbox *sandboxv1alpha1.Sandbox) error {
+	log := log.FromContext(ctx)
+	for _, pvcTemplate := range sandbox.Spec.VolumeClaimTemplates {
+		pvc := &corev1.PersistentVolumeClaim{}
+		pvcName := pvcTemplate.ObjectMeta.Name + "-" + sandbox.Name
```

Comment:
Does StatefulSet have any protection against collisions here?  Not a blocker, but we should learn from it!